tldw_Server_API/tests/Media/test_document_insights.py

```
@@ -28,13 +28,16 @@ def mock_db():
     db.get_media_by_id = MagicMock(
         return_value={"id": 1, "type": "pdf", "content": "Sample document content."}
     )
-    db.db_path_str = "/tmp/test_media.db"
+    db.db_path_str = "/tmp/test_media.db"  # nosec B108
     return db
 
 
 class _StubAdapter:
+    def __init__(self, payload=None):
```

Ruff (0.15.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)